cartography/data/jobs/cleanup/azure_import_function_apps_backups_cleanup.json

```diff
@@ -0,0 +1,15 @@
+{
+  "statements": [
+    {
+      "query": "MATCH (n:AzureFunctionAppBackup)<-[:CONTAIN]-(:AzureFunctionApp)<-[:RESOURCE]-(:AzureSubscription{id: {AZURE_SUBSCRIPTION_ID}}) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
+      "iterative": true,
+      "iterationsize": 100
+    },
+    {
+      "query": "MATCH (:AzureFunctionAppBackup)<-[r:CONTAIN]-(:AzureFunctionApp)<-[:RESOURCE]-(:AzureSubscription{id: {AZURE_SUBSCRIPTION_ID}}) WHERE r.lastupdated <> {UPDATE_TAG} WITH r LIMIT {LIMIT_SIZE} DELETE (r) return COUNT(*) as TotalCompleted",
+      "iterative": true,
+      "iterationsize": 100
+    }
+  ],
+  "name": "cleanup Azure function related resources"
```

Comment:
Please change the name to be specific. It should say `cleanup Azure Function Apps Backup resources`